src/main/java/com/epam/tc/hw3/IndexPage.java

```diff
@@ -0,0 +1,145 @@
+package com.epam.tc.hw3;
+
+import java.util.List;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class IndexPage {
+
+    @FindBy(id = "user-name")
+    private WebElement logginedUser;
+
+    @FindBy(css = "a[href='index.html']")
+    private WebElement homeItemOnHeaderSection;
+
+    @FindBy(css = "a[href='contacts.html']")
+    private WebElement contactFormItemOnHeaderSection;
+
+    @FindBy(className = "menu-title")
+    private WebElement serviceItemOnHeaderSection;
+
+    @FindBy(css = "a[href='metals-colors.html']")
+    private WebElement metalsColorsItemOnHeaderSection;
+
+    @FindBy(className = "icon-practise")
+    private WebElement firstImage;
+
+    @FindBy(className = "icon-custom")
+    private WebElement secondImage;
+
+    @FindBy(className = "icon-multi")
+    private WebElement thirdImage;
+
+    @FindBy(className = "icon-base")
+    private WebElement fourthImage;
+
+
+    @FindBy(xpath = "/html/body/div/div[2]/main/div[2]/div[2]/div[1]/div/span")
+    private WebElement firstTextUnderIcons;
+
+    @FindBy(xpath = "/html/body/div/div[2]/main/div[2]/div[2]/div[2]/div/span")
+    private WebElement secondTextUnderIcons;
+
+    @FindBy(xpath = "/html/body/div/div[2]/main/div[2]/div[2]/div[3]/div/span")
+    private WebElement thirdTextUnderIcons;
+
+    @FindBy(xpath = "/html/body/div/div[2]/main/div[2]/div[2]/div[4]/div/span")
+    private WebElement fourthTextUnderIcons;
+
+    @FindBy(id = "frame")
+    private WebElement iframeWithFrameButton;
+
+    @FindBy(id = "frame-button")
+    private WebElement iframeButton;
+
+    @FindBy(tagName = "li")
+    private List<WebElement> elementsLeftSection;
+
+    @FindBy(css = "a[href='metals-colors.html']")
+    private WebElement metalsColorsLeftSection;
+
+    /*Ex2*/
+    @FindBy(className = "menu-title")
+    private WebElement serviceOnHeaderMenu;
+
+    @FindBy(xpath = "//*[@id='mCSB_1_container']/ul/li[3]/ul/li[8]/a")
+    private WebElement differentElementsPageHeaderMenu;
+
+
+
+    public String getLoggedUser() {
+        return logginedUser.getText();
+    }
+
+    /*Ex1*/
+    public String getHomeItemOnHeaderSection() {
+        return homeItemOnHeaderSection.getText();
+    }
+
+    public String getContactFormItemOnHeaderSection() {
+        return contactFormItemOnHeaderSection.getText();
+    }
+
+    public String getServiceItemOnHeaderSection() {
+        return serviceItemOnHeaderSection.getText();
+    }
+
+    public String getMetalsColorsItemOnHeaderSection() {
+        return metalsColorsItemOnHeaderSection.getText();
+    }
+
+    public void checkFourImages() {
```

Comment:
this method does literally nothing.
here should be assertions

Reply:
fixed

Reply:
Can we have Assert in the page method?